dll/win32/shell32/CDefView.cpp

```diff
@@ -612,6 +612,33 @@ void CDefView::UpdateStatusbar()
     }
 }
 
+unsigned __stdcall CDefView::_UpdateStatusbarProc(void *args)
+{
+    CDefView* pView = static_cast<CDefView*>(args);
+    pView->AddRef();
```

Comment:
This `AddRef` is not before starting the thread, it should be before calling `_beginthreadex`, so that the object stays valid as long as the thread is alive

Reply:
Fixed.